src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+html {
+  font-family: Roboto, serif;
+  font-weight: 400;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+@import './variables';
+@import './card';
+@import './stars';
```

Comment:
Ensure that the `stars.scss` file exists and contains the necessary styles for the stars component used in your HTML.